src/jobflow_remote/remote/data.py

```diff
@@ -4,16 +4,34 @@
 import logging
 import os
 from pathlib import Path
-from typing import Any
+from typing import Any, Callable
 
 import orjson
+from jobflow.core.job import Job, JobConfig
 from jobflow.core.store import JobStore
 from maggma.stores.mongolike import JSONStore
 from monty.json import jsanitize
+from pydantic import BaseModel
 
 from jobflow_remote.utils.data import uuid_to_path
 
 
+class JobWrapper(Job):
```

Comment:
I am not sure I understand the need for this class. If it is just to have a list of arguments of `Job`'s `__init__` can't it be a list of strings instead? This will not offer any automatic update in case the `Job` API is changed, right? If we want automation maybe we can use `inspect` on the `__init__` method?

I am mainly concerned that a new developer (or me in 3 months :) )  arriving here would think this wrapper may have some functional use, while it seems to be equivalent to a list of attributes.

Reply:
The idea would be to contribute it back up to Jobflow, with the benefit being the automatic updates. That is, if we think this is useful!

Reply:
Do you mean the you would move all the properties to `Job` so to access them with `__annotations__`? Otherwise I still don't see the advantage of having a class rather than the simply a list of strings. Maybe the whole check function could be moved to jobflow? For example a `Job.required_additional_stores` method? I still need to check if this can always be determined only looking at the `Job` and not at the content of the job's `function`.

Reply:
I've just replaced the wrapper with `inspect.signature(Job)` which seems to work quite well.

Happy to migrate this to jobflow if you think that's the better place for it!

Reply:
It may be interesting to move it to jobflow, but I think we can leave it here for the time being and see if any other option or check may be needed.